packages/bpk-component-bottom-sheet/src/BpkBottomSheet.tsx

```diff
@@ -15,86 +15,145 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import { useState, type ReactNode } from "react";
+import type { ReactNode } from "react";
+import { useState } from "react";
 
-import { Portal, cssModules } from "../../bpk-react-utils";
-import { withScrim } from "../../bpk-scrim-utils";
+import BpkBreakpoint, { BREAKPOINTS } from "../../bpk-component-breakpoint";
+// @ts-expect-error Untyped import. See `decisions/imports-ts-suppressions.md`.
+import { BpkButtonLink } from '../../bpk-component-link';
+// @ts-expect-error Untyped import. See `decisions/imports-ts-suppressions.md`.
+import BpkCloseButton from '../../bpk-component-close-button';
+import { BpkDialogWrapper, cssModules } from "../../bpk-react-utils";
+import BpkNavigationBar from "../../bpk-component-navigation-bar";
 
 import STYLES from './BpkBottomSheet.module.scss';
-import BpkBottomSheetInner from "./BpkBottomSheetInner";
-
 
 const getClassName = cssModules(STYLES);
-const ScrimBpkBottomSheetInner = withScrim(BpkBottomSheetInner)
 
 export type Props = {
   actionText?: string;
+  ariaLabelledby: string;
   children: ReactNode;
   closeLabel?: string;
   closeOnEscPressed?: boolean;
   closeOnScrimClick?: boolean;
   id: string;
-  isOpen: boolean;
   onAction?: () => void;
   onClose: (
-    arg0?: TouchEvent | MouseEvent | KeyboardEvent,
+    arg0?: Event | KeyboardEvent | MouseEvent | PointerEvent | TouchEvent,
     arg1?: {
       source: 'ESCAPE' | 'DOCUMENT_CLICK';
     },
   ) => void;
   title?: string;
   wide?: boolean;
-  getApplicationElement: () => HTMLElement | null;
-  renderTarget?: null | HTMLElement | (() => null | HTMLElement);
+  isOpen: boolean;
 }
 
 const BpkBottomSheet = ({
   actionText = '',
+  ariaLabelledby,
+  children,
   closeLabel = '',
   closeOnEscPressed = false,
   closeOnScrimClick = false,
   id,
   isOpen,
   onAction = () => null,
   onClose,
-  renderTarget,
   title = '',
-  wide = false,
-  ...rest
+  wide = false
 }: Props) => {
-  const [ exiting, setExitting ] = useState(false);
+  const [exiting, setExiting] = useState(false);
+
+  const animationTimeout = 240;
+
   const handleClose = (
-    arg0?: TouchEvent | MouseEvent | KeyboardEvent,
+    timeout: number,
+    arg0?: Event | KeyboardEvent | MouseEvent | PointerEvent | TouchEvent,
     arg1?: {
       source: 'ESCAPE' | 'DOCUMENT_CLICK';
     },
-) => {
-    setExitting(true)
+  ) => {
+    setExiting(true)
     setTimeout(() => {
       onClose(arg0, arg1)
-      setExitting(false)
-    }, 240)
+      setExiting(false)
+    }, timeout)
```

Comment:
This needs to be passed in from parent because when the component is on desktop (a different animation) this gets set to 0, as seen on [lines 102](https://github.com/Skyscanner/backpack/pull/3243/files#diff-74fb49523990b42587c46681a2766595137981a62c73ad7f2cc909d181a40e5eR102) and 134

Reply:
Ohh, you're right, I hadn't realised it's a parameter of `handleClose`. Makes sense then!